src/main/java/com/example/rets_api/resource/Enums.java

```diff
@@ -78,4 +78,18 @@ public enum Operator {
         DEFAULT_ENUM_VALUE
     }
 
+    public enum CommunityType {
```

Comment:
Was a good idea, but I think will be easier to handle at the filter if we keep using as indicators, as we did in LotData or we are doing in viewData. I'm changing the get in filter also and the indicator approach will help me with this change :)

Reply:
done